---
string hostname
string lidar_mode
string prod_line
float64[] beam_azimuth_angles
float64[] beam_altitude_angles
float64[] imu_to_sensor_transform
float64[] lidar_to_sensor_transform